server/src/main/java/org/elasticsearch/action/support/ActiveShardCount.java

```diff
@@ -175,7 +174,7 @@ public boolean enoughShardsActive(final IndexShardRoutingTable shardRoutingTable
             // which gives us the total number of shard copies
             return activeShardCount == shardRoutingTable.replicaShards().size() + 1;
         } else if (this == ActiveShardCount.DEFAULT) {
-            return activeShardCount >= 1;
+            return activeShardCount >= 1 && shardRoutingTable.getActiveSearchShardCount() >= 1;
```

Comment:
nit: if there's ≥1 active search shard then there's ≥1 active shard

```suggestion
            return shardRoutingTable.getActiveSearchShardCount() >= 1;
```